src/routes/write-exercise/services/AbstractWriteExerciseService.ts

```diff
@@ -14,7 +15,8 @@ export default abstract class AbstractWriteExerciseService {
     public navigation: StackNavigationProp<RoutesParams, 'WriteExercise'>,
     public setCurrentIndex: React.Dispatch<React.SetStateAction<number>>,
     public setIsAnswerSubmitted: React.Dispatch<React.SetStateAction<boolean>>,
-    public setVocabularyItemWithResults: React.Dispatch<React.SetStateAction<VocabularyItemResult[]>>, // eslint-disable-next-line no-empty-function
+    public setVocabularyItemWithResults: React.Dispatch<React.SetStateAction<VocabularyItemResult[]>>,
+    public storageCache: StorageCache, // eslint-disable-next-line no-empty-function
```

Comment:
```suggestion
    public storageCache: StorageCache,
    // eslint-disable-next-line no-empty-function
```
Just to make it clear at first glance that the eslint problem is in the next line, not the storage one (I know you didn't write it, but it did confuse me 😅 )